create-aleo-app/template-offline-public-transaction-ts/src/index.ts

```diff
@@ -93,7 +93,9 @@ async function buildBondingTxOffline(recipientAddress: Address, amount: number,
     }
 
     const bondTx = <Transaction>await programManager.buildBondPublicTransaction(
-        recipientAddress.to_string(),
+        stakerAddress.to_string(),
```

Comment:
Since we're making breaking changes anyways, I think it would be nice if it used an options object, instead of using multiple arguments.

Reply:
I agree, but don't currently have the allocation to do so? Would you be able to make that change? Or maybe @onetrickwolf?

Reply:
Sure, I can do that in another PR.

Reply:
In the case of the method that mirror credits.aleo, I'd like to keep this as is so that users ensure that users are considering the arguments going into Aleo instructions.